backend/src/main/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProvider.java

```diff
@@ -32,7 +31,7 @@
     @RestClient
     RbacServiceToService rbacServiceToService;
 
-    @ConfigProperty(name = "recipient-provider.rbac.elements-per-page", defaultValue = "40")
+    @ConfigProperty(name = "recipient-provider.rbac.elements-per-page", defaultValue = "1000")
```

Comment:
I though we were going to use a higher number based on Jay's comments.
Anyhow we can tweak this number based on the data that we collect.

Reply:
Yeah we could go higher from the start but I'd rather see the results with that one and then adjust with the configuration.